corehq/apps/dropbox/models.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from dropbox.client import DropboxOAuth2Flow, DropboxClient
+
+
+class DropboxUploader(object):
```

Comment:
this will eventually be an actual django model. i'm thinking we can just save a minimal model that records the progress of the upload, size, and destination just to keep records